packages/cli/src/commands/datasources/push/actions.ts

```diff
@@ -11,10 +11,14 @@ export const pushDatasource = async (space: string, datasource: SpaceDatasource)
   try {
     const client = mapiClient();
 
-    const { data } = await client.post<{
-      datasource: SpaceDatasource;
-    }>(`spaces/${space}/datasources`, {
-      body: JSON.stringify(datasource),
+    const { data } = await client.datasources.create({
+      path: {
+        space_id: Number.parseInt(space),
+      },
+      body: {
+        datasource,
```

Comment:
Hi @imranolas I'm getting a type error in the body of a couple of create methods, but only in the create, update methods are fine. 
<img width="819" height="428" alt="Screenshot 2025-09-02 at 08 40 44" src="https://github.com/user-attachments/assets/f4d19070-7cbc-43ba-83ad-c8664a825026" />


<img width="892" height="405" alt="Screenshot 2025-09-02 at 08 38 07" src="https://github.com/user-attachments/assets/585acc10-f2f8-48e9-b833-3566fc0519b2" />